hexa/datasets/queue.py

```diff
@@ -90,6 +117,9 @@ def generate_dataset_file_sample_task(
                 replace=True,
             )
             dataset_file_metadata.sample = file_sample.to_json(orient="records")
+            dataset_file_metadata.profiling = json.dumps(
```

Comment:
Not sure you need to dumps the json since you defined the column as a JSON field. I would hope they do that for us.

Reply:
Still need it, tested in tests